ArduPlane/Parameters.cpp

```diff
@@ -644,14 +644,14 @@ const AP_Param::Info Plane::var_info[] = {
     // @User: Advanced
     ASCALAR(min_gndspeed_cm,      "MIN_GNDSPD_CM",  MIN_GNDSPEED_CM),
 
-    // @Param: TRIM_PITCH_CD
+    // @Param: PITCH_TRIM
     // @DisplayName: Pitch angle offset
     // @Description: Offset applied to AHRS pitch used for in-flight pitch trimming. Correct ground leveling is better than changing this parameter.
-    // @Units: cdeg
-    // @Range: -4500 4500
+    // @Units: deg
+    // @Range: -45 45
     // @Increment: 10
```

Comment:
Need to update (or remove) the increment.

Reply:
Removed